recipes/kmedoids/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "kmedoids" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kno10/python-kmedoids/archive/refs/tags/v{{ version }}.tar.gz
+  #  `openssl sha256 <file name>`.
+  sha256: d334848ecc0d4b17c0e041b975f29b1963cd64e7c3743d9b9ec157ae5780e2c2
+
+build:
+  number: 0
+  script:
+    - export CARGO_NET_GIT_FETCH_WITH_CLI=true
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+    - numpy
+  host:
+    - python
+    - pip
+    - maturin
+  run:
+    - python
+    - numpy
+    - libgcc-ng  # [linux]
```

Comment:
it is really strange that you need this particular line. Are you missing a dependency on the c compiler?

Reply:
It failed to build in the docker container without this.
IMHO this should be inferred from the rust compiler dependency.
I have just pushed without this line, so you can see the build logs and maybe you have a better solution.

Reply:
You can see the error at
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=615973&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823

Reply:
cc: @conda-forge/help-rust <-- hmm this doesn't seem to exit.

cc: @conda-forge/help-c-cpp any ideas?

Reply:
This (the rust build process) probably calls at one stage to a C compiler. As it is transitive, it builds but we need to specify the dependency explicitly for the `run_exports` to work.

Reply:
So is it fine as it is, or am I supposed to pull in the C compiler/dependency using
```
- {{ compiler('c') }}
- {{ compiler('rust') }}
```
which seems more portable?

Reply:
I would opt for:
```
- {{ compiler('c') }}
- {{ compiler('rust') }}
```
since it will keep the correct dependency exported.

Without it, the pinning may be to an inconsistent version.

Reply:
The version with `- {{ compiler('c') }}` seems to build, and now no longer needs the `#  [linux]` extra dependency. I flag this as resolved.